builder/image-ros.sh

```diff
@@ -187,6 +187,10 @@ catkin_make run_tests && catkin_test_results
 echo_stamp "Change permissions for catkin_ws"
 chown -Rf pi:pi /home/pi/catkin_ws
 
+echo_stamp "Set the setuid bit for ws281x led strip driver"
```

Comment:
SUID bits should be set in the package itself. The way it's done here is very fragile.

Reply:
Ok, but this is not so trivial to do. So I propose to merge it like this, and then transfer setting `setuid` to the package, if we have time.

Reply:
Your proposition leaves us with a broken package (that won't work without additional steps by the user that involve changing the package files, which is very much discouraged and goes against the idea of a package-based distribution) and a fragile installation script. It is better to just pull the `ros_led` package sources and build them along with everything else than to do what is proposed.

Reply:
I've uploaded the `ws281x` package that sets the SUID bit. These lines should no longer be necessary.